CONTRIBUTING.md

```diff
@@ -42,29 +44,53 @@ hooks if needed. When installed the `pre-commit` hooks will be run on every `git
-| `./do Dev:LintFix`     | Tries to fix linter issues.                                                               |
-| `./do Dev:Unit`        | Runs local unittests.                                                                     |
+| Command                | Description                                                                                                                     |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| `./do Dev:Create`      | Sets up the local development cluster.                                                                                          |
+| `./do Dev:Destroy`     | Deletes the local development cluster.                                                                                          |
+| `./do Dev:Generate`    | Generate code, api docs, install files.                                                                                         |
+| `./do Dev:Integration` | Runs local integration tests in a KinD cluster. (Run `Dev:Destroy` before to ensure that you're testing a freshly compiled PKO) |
+| `./do Dev:Lint`        | Runs local linters to check the codebase.                                                                                       |
+| `./do Dev:LintFix`     | Tries to fix linter issues.                                                                                                     |
+| `./do Dev:Unit`        | Runs local unittests.                                                                                                           |
+| `./do Dev:Run`         | Prepares development cluster and `go runs` package-operator-manager out-of-cluster.                                             |
+
+#### Setting up the local development cluster without shadowing image pulling (pod workload images and package-operator package images) from quay.io
+
+By default, the local development environment will redirect image pulls from `quay.io` to the local dev registry running at `localhost:5000` instead.
 
+These redirects affect:
+- package-operator-manager: will pull package images when a `(Cluster)Package` object changes.
+- kubelet/container-runtime: will pull workload images specified in `Pod` objects.
 
-#### Accessing a cluster deployed using dev:deploy
+If you need to access (package) images from `quay.io` you can move the override out of the way by:
+1. Ensuring to fully destroy your current development environment, by running: `./do Dev:Destroy`.
+2. Recreating the development environment with another registry prefix (that will be redirected to localhost:5000, which in turn frees up access to quay.io): `IMAGE_REGISTRY=ctr.package-operator.run/dev ./do Dev:Create`.
+3. Prepend all other cardboard commands with `IMAGE_REGISTRY=ctr.package-operator.run/dev` (or export it via an `.envrc`).
+
+We're thinking of changing this away from quay.io by default.
```

Comment:
Should this be here? Seems more like a personal note/todo.

Reply:
> We're thinking of changing this away from quay.io by default.

this line alone?

It's there for trying to invite criticism of this default and to communicate that we're open to chaning it, if one wants to complain about it.
But I can remove it as well.

Reply:
Alright, I'm fine either way, no need to change.